src/components/SavedCryptos.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import CryptoCard from "./CryptoCard";
+import axios from "axios";
+import { useQueries } from "@tanstack/react-query";
+import CoinGeckoResponse from "../models/CoinGeckoResponse";
+import SavedCrypto from "../models/SavedCrypto";
+import "./SavedCryptos.css";
+
+const fetchCryptoData = async (coinId: string) => {
+  const response = await axios.get<CoinGeckoResponse>(
+    `https://api.coingecko.com/api/v3/simple/price?ids=${coinId}&vs_currencies=usd,eur,gbp&precision=2`
+  );
+  return response.data[coinId];
+};
```

Comment:
is this being repeated?

Reply:
Yes, an overlook from my side. Will update the code structure to make this reusable